modules/choose/Chooser.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+module.exports = class Chooser {
+  /**
+    * This will throw if it's in a locked channel.
+    */
+  async chooseRandomUser(bot, channel) {
+    const channelData = await bot.getChannelById(data.channel);
+    if (!channelData.members) {
+      return false;
+    }
+
+    const randUser = this.getRandomUser(channelData.members);
+    return randUser; 
+  }
+
+  getRandomUser(members) {    
+    return members[
+      Math.floor(Math.random() * members.length)
```

Comment:
Dunno what the execution environment for this plugin is exactly but make sure you've seeded the PRNG beforehand at some point otherwise your pairing could get weird.

Reply:
You can't seed the PRNG in JavaScript you nerd